src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -508,7 +512,7 @@ namespace
     }
 }
 
-
+bool UsersConfigAccessStorage::ALLOW_PLAINTEXT_AND_NO_PASSWORD=true;
```

Comment:
Static variables like this are always bad, because they produce data race easily.

Please use member variable with mutex or atomic instead, and place it in the `AccessControl` class, not to `UsersConfigAccessStorage`.